tests/bdd-smoke/utils/command_runner.go

```diff
@@ -0,0 +1,76 @@
+package utils
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+	"time"
+
+	expect "github.com/Netflix/go-expect"
+	"github.com/hinshun/vt10x"
+)
+
+// RunOSCommandWithArgs executes a command in the operating system with arguments
+func RunOSCommandWithArgs(command string, arguments []string, path string) string {
+	cmd := exec.Command(command, arguments...)
+	cmd.Dir = path
+	cmd.Stdin = strings.NewReader("")
+	var outb, errb bytes.Buffer
+	cmd.Stdout = &outb
+	cmd.Stderr = &errb
+	err := cmd.Run()
+	if err != nil {
+		fmt.Println("Error: failed running the command \"" + command + "\" with the arguments:")
```

Comment:
nit: this could be more robust and compact with:

```go
fmt.Printf("Error: failed running the command %q with the arguments:", command)
```

although printing straight to the terminal feels strange anyway.  Does that get captured somewhere?  Should we be logging these instead, or attaching it to the error before passing it to `log.Fatal`?